CONTRIBUTING.md

```diff
@@ -269,6 +269,19 @@ manage ingest_bulk_cve --subset 100
 This will take a few minutes on an average machine.
 Not passing `--subset N` will take about an hour and produce ~500 MB of data.
 
+### Caching suggestions and issues
+
+Fetching information about individual suggestions or published issues is very frequent when browsing through the security tracker. For each view requested, this would typically involve database queries that would take way too long to execute.
+
+Therefore, we pre-execute these queries and cache the results on every suggestion and published issue. The cached information is used when navigating in the tracker. A django hook on inserting or updating suggestions or published issues ensures caching for new elements but the operation needs to be performed once for pre-existing elements in the database.
```

Comment:
Please move this into the code, it's important information but not relevant for using the system. We need to watch out not to overload readers who just need to get the job done here.

Reply:
Yeah yeah the instructions should be here. Just not the explanation on why things are built the way they are! That's for in-code comments :)